src/Employee/EmployeeFail.java

```diff
@@ -0,0 +1,46 @@
+package Employee;
+
+import java.util.Random;
+
+public class EmployeeFail {
+    private String SNP;
+    private String post;
+    private String email;
+    private String telephone;
+    private int salary;
+    private int age;
+
+
+    public EmployeeFail(String SNP, String post, String email, String telephone, int salary) {
+        this.SNP = SNP;
+        this.post = post;
+        this.email = email;
+        this.telephone = telephone;
+        this.salary = salary;
+        this.age = ageRandom();
+
+
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public int ageRandom() {
+        return new Random().nextInt(15)+30;
```

Comment:
как вы заметили, рандом лучше увеличить
```suggestion
        return new Random().nextInt(30)+30;
```